docs/diagrams/MenuFeaturesSequenceDiagram.puml

```diff
@@ -0,0 +1,33 @@
+@startuml
+'https://plantuml.com/sequence-diagram
```

Comment:
Good use of UML sequence and class diagrams to explain the working of the system in a simple way.